<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Num. of Trial:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumTrial">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1200</width>
      <height>450</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTraining">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Training</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTesting">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Testing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_groundtruth">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ground truth</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_target">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>530</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Target trajectory</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_preceding">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Preceding trajectory</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_lead">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>590</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Lead trajectory</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>label2</zorder>
   <zorder>btnStart</zorder>
   <zorder>lblFileName</zorder>
   <zorder>lblNumTrial</zorder>
   <zorder>btnTraining</zorder>
   <zorder>btnTesting</zorder>
   <zorder>btnReset</zorder>
   <zorder>chb_groundtruth</zorder>
   <zorder>chb_target</zorder>
   <zorder>chb_preceding</zorder>
   <zorder>chb_lead</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionOpenAll"/>
    <addaction name="action_Save"/>
    <addaction name="actionOpenThree"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionOpenAll">
   <property name="text">
    <string>Open&amp;All</string>
   </property>
  </action>
  <action name="actionOpenThree">
   <property name="text">
    <string>OpenThree</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Open</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ResetButtonEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>StartButtonEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>800</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTesting</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>TestingButtonEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTraining</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>TrainingButtonEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Save</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SaveDataPerTrial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenAll</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenAllFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenThree</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenThreeFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_groundtruth</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChkBoxGroundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_target</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChkBoxTargetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_preceding</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChkBoxPrecedingClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chb_lead</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChkBoxLeadClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>ResetButtonEntered()</slot>
  <slot>StartButtonEntered()</slot>
  <slot>TrainingButtonEntered()</slot>
  <slot>TestingButtonEntered()</slot>
  <slot>SaveDataPerTrial()</slot>
  <slot>OpenAllFile()</slot>
  <slot>OpenThreeFiles()</slot>
  <slot>ChkBoxGroundClicked()</slot>
  <slot>ChkBoxTargetClicked()</slot>
  <slot>ChkBoxPrecedingClicked()</slot>
  <slot>ChkBoxLeadClicked()</slot>
 </slots>
</ui>
